### DISCUSSION TITLE
同步错误：remote is modified (branch 9)but size larger than 20000000,don't know what to do:

### DISCUSSION THREAD

**Discussion by yuanhunguiguli** [remotely-save/remotely-save#748] [Q&A]
最新版本使用s3在window上同步过一次，但在另外一台设备上（已经有原库一些文件）同步失败，错误提示：remote is modified (branch 9)but size largerthan 20000000,don't know what to do:

**Comment by fyears** [comment]
你限制了不超过20mb，但是现在一端超过了一端没超过，因此插件不知道怎么办。可以考虑文件大小放宽限制来同步
